pkg/agent/cniserver/ipam/ipam_service.go

```diff
@@ -26,12 +26,18 @@ import (
 
 var ipamDrivers map[string]IPAMDriver
 
-type IPAMConfig struct {
-	Type    string `json:"type,omitempty"`
-	Subnet  string `json:"subnet,omitempty"`
+type Range struct {
```

Comment:
Sorry, why Subnet + Gateway is called "Range"?

Reply:
Actually, I am following host-local IPAM fields naming here, see:
https://github.com/containernetworking/plugins/blob/master/plugins/ipam/host-local/backend/allocator/config.go#L50
https://github.com/containernetworking/plugins/blob/master/plugins/ipam/host-local/backend/allocator/config.go#L65

Reply:
Ok.